assets/js/components/data/index.js

```diff
@@ -186,22 +183,7 @@ const dataAPI = {
 			return;
 		}
 
-		let addedNoticeCount = 0;
-
-		// Add insufficient scopes warning.
-		if ( [ 'authError', 'insufficientPermissions' ].includes( data.reason ) ) {
-			addFilter( 'googlesitekit.ErrorNotification',
-				'googlesitekit.AuthNotification',
-				fillFilterWithComponent( UnsatisfiedScopesAlert ), 1 );
-			addedNoticeCount++;
-		}
-
-		if ( data.reconnectURL ) {
-			addFilter( 'googlesitekit.ErrorNotification',
-				'googlesitekit.AuthNotification',
-				fillFilterWithComponent( AuthError ), 1 );
-			addedNoticeCount++;
-		}
```

Comment:
@eugene-manuilov – `AuthError` and `UnsatisfiedScopesAlert` are already present in the new `ErrorNotifications` component which will be the only version of these going forward, however I don't think `setAuthError` is currently called for legacy data requests. Is this handled somewhere?

Reply:
Yes, it's handled here: https://github.com/google/site-kit-wp/blob/f538d84956e04a1b9ef2650da6570b6cc0fe97d9/assets/js/googlesitekit/api/index.js#L88-L101

Reply:
@eugene-manuilov this only covers requests made with the newer API module. The removed code above was specific to the legacy dataAPI which is still used until removed in #2258.

Reply:
Maybe this is okay if we're not concerned about the legacy API any more. @felixarntz ?

Reply:
@felixarntz – do we need to ensure these are still handled for legacy data requests?

Reply:
Let's remove those pieces, we shouldn't need to worry about the legacy data API anymore due to #2258. However, I think it's reasonable to say we shouldn't ship this as a "broken" state. I'm going to assign this issue as well as #2258 to the 1.38.0 release, so that we can tie them together. In other words, let's hold off merging this PR until we branched off 1.37.0 into `main`.